scripts/install.sh

```diff
@@ -12,11 +12,22 @@ BREW=$(which brew)
 
 set -e
 
+function copy_binary() {
+  if [[ ":$PATH:" == *":$HOME/bin:"* ]]; then
```

Comment:
i think i'd slightly prefer .local/bin because there's a formal spec around it?
https://www.freedesktop.org/software/systemd/man/file-hierarchy.html